include/edm4hep/GenToolInfo.h

```diff
@@ -0,0 +1,44 @@
+#ifndef EDM4HEP_GENTOOLINFO_H
+#define EDM4HEP_GENTOOLINFO_H
+
+#include "podio/Frame.h"
+#include "Constants.h"
+#include <string>
+#include <vector>
+
+namespace edm4hep {
+
+struct GenToolInfo {
+  std::string name;
+  std::string version;
+  std::string description;
+};
+
+const std::vector<GenToolInfo> getGenToolInfos(podio::Frame& frame) {
+  auto toolInfos = std::vector<GenToolInfo>();
+  auto names = frame.getParameter<std::string>("name");
+  auto versions = frame.getParameter<std::string>("version");
+  auto descriptions = frame.getParameter<std::string>("description");
+  for (unsigned int i = 0; i < names.size(); ++i) {
+    toolInfo.emplace_back(names[i], versions[i], descriptions[i]);
```

Comment:
```suggestion
    toolInfos.emplace_back(names[i], versions[i], descriptions[i]);
```
missing s (or extra s above)